src/plot/series/mark-series.js

```diff
@@ -32,6 +33,35 @@ const predefinedClassName = 'rv-xy-plot__series rv-xy-plot__series--mark';
 const DEFAULT_STROKE_WIDTH = 1;
 
 class MarkSeries extends AbstractSeries {
+  shouldComponentUpdate(nextProps, nextState) {
+    return [
+      [this.props.animation, nextProps.animation],
+      [this.props.className, nextProps.className],
+      [this.props.colorDomain, nextProps.colorDomain],
+      [this.props.colorRange, nextProps.colorRange],
+      [this.props.data, nextProps.data],
+      [this.props.fillDomain, nextProps.fillDomain],
+      [this.props.fillRange, nextProps.fillRange],
+      [this.props.innerHeight, nextProps.innerHeight],
+      [this.props.innerWidth, nextProps.innerWidth],
+      [this.props.marginLeft, nextProps.marginLeft],
+      [this.props.marginTop, nextProps.marginTop],
+      [this.props.nullAccessor, nextProps.nullAccessor],
```

Comment:
what is this?

Reply:
javascripts

Reply:
specifically - nullAccessor? where does that come from?

Reply:
Oh. That's an optional prop which returns a boolean but for `MarkSeries` its defined at the very bottom of the file 
```javascript
MarkSeries.defaultProps = {
  nullAccessor: () => true
};
```
and from the docs
```markdown
#### nullAccessor (optional)
Type: `function`  
Default: `null`  
A function that will be invoked for each data element that will return a boolean that specifies if the data point should be drawn or not. For more information see [the D3 documentation](https://github.com/d3/d3-shape#line_defined).
```